test/spec/attribution_control.js

```diff
@@ -0,0 +1,29 @@
+describe('L.control.attribution', function() {
+    'use strict';
+    it('toggles leaflet-compact-attribution class', function() {
+        var el = document.querySelector('body').appendChild(document.createElement('div'));
```

Comment:
The added element should be removed after the test completes, so that the DOM is clean for future tests. [example](https://github.com/mapbox/mapbox.js/blob/db25bba68d2fb55b549d2f23988cebc31a95bc5e/test/spec/share_control.js#L5-15)